src/io/flutter/actions/OpenInAndroidStudioAction.java

```diff
@@ -30,11 +30,14 @@
 
 public class OpenInAndroidStudioAction extends AnAction {
 
-  private static void openFileInStudio(@NotNull VirtualFile file, @NotNull String androidStudioPath) {
+  private static void openFileInStudio(@NotNull VirtualFile file, @NotNull String androidStudioPath, @Nullable String sourceFile) {
```

Comment:
For understandability, I wonder if we couldn't change `file` to `projectFile`?

Reply:
Sure.